app/services/vimeo/api_service.rb

```diff
@@ -62,7 +58,7 @@ def execute(uri, request)
       net_http.use_ssl = true
       raw_response = net_http.request(request)
 
-      JSON.parse(raw_response.body).with_indifferent_access
+      JSON.parse(raw_response.body).with_indifferent_access unless raw_response.body.nil?
```

Comment:
This would cause the method to return two different _classes_ - `nil` when there is no response body, and `ActiveSupport::HashWithIndifferentAccess`, when there is a response. It's always safer for a method to have only one response _type_.

```suggestion
      if raw_response.body.present?
        JSON.parse(raw_response.body)
      else
        {}
      end.with_indifferent_access
```